site/src/components/UsersTable/UsersTable.stories.tsx

```diff
@@ -28,3 +28,10 @@ Empty.args = {
   users: [],
   roles: MockSiteRoles,
 }
+
+export const Loading = Template.bind({})
+Loading.args = {
+  users: [],
+  roles: MockSiteRoles,
+  isLoading: true,
+}
```

Comment:
I like this story since it tests a particular behavior of the component, but I am not sure if it is a good UI snapshot since the loader is spinning and the snapshot may change across different builds. 🤔

Reply:
Good thought. I haven't read through this in detail, but could we pause the animation for the loader? Or introduce a slight delay? https://www.chromatic.com/docs/animations

Reply:
Thanks for pointing to this, I used
```ts
Loading.parameters = {
  chromatic: { pauseAnimationAtEnd: true },
}
```
and it seems to be constant for at least 3 builds.